internal/pkg/manifest/worker_svc.go

```diff
@@ -38,18 +45,76 @@ type WorkerServiceConfig struct {
 type WorkerServiceProps struct {
 	WorkloadProps
 	HealthCheck *ContainerHealthCheck // Optional healthcheck configuration.
-	Topics      *[]TopicSubscription  // Optional topics for subscriptions
+	Topics      []TopicSubscription   // Optional topics for subscriptions
 }
 
 // SubscribeConfig represents the configurable options for setting up subscriptions.
 type SubscribeConfig struct {
-	Topics *[]TopicSubscription `yaml:"topics"`
+	Topics []TopicSubscription `yaml:"topics"`
+	Queue  *SQSQueue           `yaml:"queue"`
 }
 
 // TopicSubscription represents the configurable options for setting up a SNS Topic Subscription.
 type TopicSubscription struct {
-	Name    string `yaml:"name"`
-	Service string `yaml:"service"`
+	Name    string    `yaml:"name"`
+	Service string    `yaml:"service"`
+	Queue   *SQSQueue `yaml:"queue"`
+}
+
+// SQSQueue represents the configurable options for setting up a SQS Queue.
+type SQSQueue struct {
+	Name       *string          `yaml:"name"`
+	Retention  *string          `yaml:"retention"`
+	Delay      *string          `yaml:"delay"`
+	Timeout    *string          `yaml:"timeout"`
+	KMS        *bool            `yaml:"kms"`
+	DeadLetter *DeadLetterQueue `yaml:"dead_letter"`
+	FIFO       *FIFOOrBool      `yaml:"fifo"`
+}
```

Comment:
let's use `*time.Duration` to represent these. This way customers can unmarshal anything from `3600s` to `4d6h` and the duration package will parse it correctly. 

Check out `convertHealthCheck` for examples of unmarshaling and dealing with `duration` objects correctly.
```suggestion
type SQSQueue struct {
	Name       *string          `yaml:"name"`
	Retention  *time.Duration          `yaml:"retention"`
	Delay      *time.Duration          `yaml:"delay"`
	Timeout    *time.Duration          `yaml:"timeout"`
	KMS        *bool            `yaml:"kms"`
	DeadLetter *DeadLetterQueue `yaml:"dead_letter"`
	FIFO       *FIFOOrBool      `yaml:"fifo"`
}
```